docs/user/dashboard/create-visualizations.asciidoc

```diff
@@ -0,0 +1,306 @@
+= Panels and visualizations
```

Comment:
I think in the long run in the new docs system it sounds like we'll want to separate the navigation from the content and each page to just start with an H1. So in that case this page would be split into multiple pages and the includes with leveloffsets would exist in the nav file.  Not necessary at this moment, but I think worth considering for future restructuring.

Reply:
100% agree. While working on this PR I've started to move in that direction (have more files start with H1 and play with leveloffset instead in includes). This PR was already too big and contains content updates as well but it's probably worth planning for it, as it will also make maintenance and contributions easier to approach.